versions.props

```
@@ -15,6 +15,8 @@ com.googlecode.java-diff-utils:diffutils = 1.3.0
 com.puppycrawl.tools:checkstyle = 10.3.2
 org.checkerframework:* = 3.23.0
 com.palantir.gradle.utils:* = 0.1.0
+com.uber.nullaway:nullaway = 0.10.1
+org.checkerframework:dataflow-nullaway = 3.25.0
```

Do we need this more specific version pin compared to above?

```suggestion
org.checkerframework:dataflow-nullaway = 3.25.0
```